modules/parser/src/parsers/parse-xviz-message-sync.js

```diff
@@ -37,15 +39,17 @@ export function parseXVIZMessageSync(message, onResult, onError, opts) {
     return;
   }
 
+  const {messageType} = opts;
+
   try {
-    const xvizData = new XVIZData(message);
+    const xvizData = new XVIZData(message, {messageType});
```

Comment:
I am thinking we could just pass `opts` through there.  It's a little broad, but what means is we need to be carefull with XVIZData() and parseXVIZData(). as we would be passing opts through both.

Alternatively , we could scope it in the opts, like `opts = {dataOptions = {}, parseOptions = {}}`

Reply:
updated